source/02-EV/01.ptx

```diff
@@ -12,12 +12,19 @@
 
 <subsection>
 <title>Warm Up</title>
-<!-- these numbers are made up; have a look and see if I want a more intentional choice here. -->
 <activity>
   <statement>
     <p>
-      Check whether or not the Euclidean vector <m>\vec{v}=\left[\begin{array}{c} 1 \\ -1 \\ 2 \end{array}\right]</m> is a solution to the given vector equation:
-      <me>x_1\left[\begin{array}{c} -1 \\ 2 \\ 3 \end{array}\right]+x_2\left[\begin{array}{c} 2 \\ -1 \\ 0 \end{array}\right]+x_3\left[\begin{array}{c} 1 \\ -1 \\ 1 \end{array}\right]=\left[\begin{array}{c} 1 \\ 0 \\ 1 \end{array}\right]</me>
+Discuss which of the vectors 
+<m>\vec{u}=\left[\begin{array}{c} 1 \\ -1 \\ 2 \end{array}\right]</m> and 
+<m>\vec{v}=\left[\begin{array}{c} 0 \\ 3 \\ -1 \end{array}\right]</m>
+is a solution to the given vector equation:
+      <me>
+x_1\left[\begin{array}{c} -1 \\ 2 \\ 3 \end{array}\right]+
+x_2\left[\begin{array}{c} 2 \\ -1 \\ 0 \end{array}\right]+
+x_3\left[\begin{array}{c} 1 \\ -1 \\ 1 \end{array}\right]=
+\left[\begin{array}{c} -1 \\ 1 \\ 5 \end{array}\right]
+      </me>
```

Comment:
I like having one that works and one that doesn't, giving students some scaffolding that lets them fix silly mistakes caused by arithmetic or fat fingers ("wait I'm getting both aren't solutions, did I mess something up?")